user/languages/android.md

```diff
@@ -179,9 +179,10 @@ This can be overridden as described in the [general build configuration](/user/c
 
 A peculiarity of dependency caching in Gradle means that to avoid uploading the cache after every build you need to add the following lines to your `.travis.yml`:
 
-```
+```yaml
 before_cache:
-  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
```

Comment:
Copied from Java which should be the same: https://docs.travis-ci.com/user/languages/java/#Caching.